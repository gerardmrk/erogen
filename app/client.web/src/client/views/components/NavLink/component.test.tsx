describe("<NavLink/>", () => {
  it("renders ok", async () => {
    expect(true).toBe(true);
  });
});
