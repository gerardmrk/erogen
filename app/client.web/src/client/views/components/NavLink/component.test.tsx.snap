// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<NavLink/> snapshot 1`] = `
<a
  aria-current="page"
  className="active"
  href="/"
  onClick={[Function]}
  style={Object {}}
>
  <span>
    Home
  </span>
</a>
`;
